### ISSUE TITLE
clearCache was not declared

### ISSUE THREAD

**Issue by sophana** [Andersbakken/rtags#1326]
Hi

can't compile on ubuntu bionic, with clang-8
error is: 
rtags/src/IndexDataMessage.h:103:9: error: ‘clearCache’ was not declared in this scope

git grep clearCache give only one result.
This line was added in march 2019.

I commented the line, but I'm not sure if this is a good solution.

**Comment by Klump3n** [comment]
So I had the same issue. I installed rtags ages ago and it worked. Updated emacs/spacemacs and it wanted a newer rtags version. Located old rtags dir, did a git pull && make clean && cmake . (with that export option) && make and this failed with the same error.

Fixed it by getting a fresh instance of the repository, then getting [rct](https://github.com/Andersbakken/rct) too (putting it into src/, I also did a cmake on rct but I don't know if that's necessary) and then also getting libcppunit-dev from the debian repo.
Now it compiles without errors.
